### DISCUSSION TITLE
How to use yarn version in .yarn/releases

### DISCUSSION THREAD

**Discussion by wdfinch** [yarnpkg/berry#2811] [Q&A] [answered]
Hi all I had a quick question regarding the yarn version I'm using. When I run the yarn command inside my project directory, it's using the one I have installed outside the project

```
➜ which yarn                            
/usr/bin/yarn
```

The yarn bin is present in `.yarn/releases/yarn-3.0.0-rc.2.cjs`. I would like to use this version so there is no difference between the version I am using and the version other team members are using. What's the recommended way to do this? Is there some yarn setting I can use or do I need to manually alias the `yarn` command to that file?

An example would be much appreciated.

**Comment by merceyz** [comment, marked as answer]
That's what the `yarnPath` setting does, it's set automatically whenever you run `yarn set version ...` https://yarnpkg.com/configuration/yarnrc#yarnPath

**Reply by wdfinch** [reply]
Thanks for the reply. Ok, that makes sense. I didn't see that config setting in the docs. Two follow up questions:

1. When I run the `yarn set version ...` command, the `yarnPath` is not automatically added to my `.yarnrc.yml` file. The bin is added in `./.yarn/releases`. Is this a bug or do you manually need to add this path in?

2. After setting this path manually and running `yarn install`, `which yarn` still outputs `/usr/bin/yarn`. Therefore should I assume that while the yarn I have in bin is running it immediately yields control to the version specified by `yarnPath`?

I set my `yarnPath` to `yarnPath: ./.yarn/releases/yarn-3.0.0-rc.2.cjs` as this is standard install location.

**Reply by merceyz** [reply]
> Is this a bug or do you manually need to add this path in?

It is supposed to update it automatically

> Therefore should I assume that while the yarn I have in bin is running it immediately yields control to the version specified by yarnPath?

That is correct

**Reply by wdfinch** [reply]
Thanks!
